### ISSUE TITLE
connect directly to node in BeamWallet?

### ISSUE THREAD

**Issue by BrannonKing** [BeamMW/cuda-miner#9]
Can I connect the Cuda miner directly to the BeamWallet node (on its listener port)? I just wanted to mine locally. However, when I attempt it I get the error "unknown protocol".

**Comment by emkman** [comment]
@BrannonKing yes you must use the stratum port not the node port. This is all covered in the guide: https://beam-docs.readthedocs.io/en/latest/rtd_pages/user_mining_beam.html#mining-using-external-miner 
Github is only for bugs not questions about usage.
